spec/models/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it { is_expected.to validate_uniqueness_of(:screen_name) }
+  it { is_expected.to validate_presence_of(:screen_name) }
+  it { is_expected.to validate_presence_of(:name) }
+
+  let(:user_a) { create(:user) }
+  let(:user_b) { create(:user, screen_name: :capybot) }
+
+  describe '#follow!' do
+    before { user_a.follow!(user_b) }
+
+    it { expect(user_a.following).to include(user_b) }
+    it { expect(user_b.followers).to include(user_a) }
+  end
+
+  describe '#unfollow!' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.